### PR TITLE
first commit for CiseScope

### PR DESCRIPTION
**Pull request by pr-Dutta**
# Pull Request

## Description

Please include a summary of the changes you have made. Highlight the key aspects of your project.

## Checklist

- [ ] My project includes a `README.md` file with a brief overview of the project, how to set it up, and any relevant information.
- [ ] I have added a screenshot or screen recording showing the functionality of the app.
- [ ] I have tested the project and confirmed it works as expected.

## Screenshots

If applicable, please add a screenshot or a screen recording to demonstrate the functionality of your app.

## Project README

Please ensure that your project's `README.md` is detailed and includes:

- A brief introduction to the project.
- Installation steps.
- Key features.
- Any additional instructions or information.

## Additional Notes

Is there anything else the maintainers should know about your contribution?

---

Thank you for your contribution!

### CODE REVIEW

File: .gitignore
Comment (anandwana001) [inline .gitignore:1]: remove parent file

File: app/src/main/java/com/example/cinescope/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import com.example.cinescope.ui.theme.CineScopeTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            CineScopeTheme {
+                Scaffold(modifier = Modifier.fillMaxSize()) { _ ->
+                    CineScopeApp()
```
Comment (anandwana001) [inline app/src/main/java/com/example/cinescope/MainActivity.kt:23]: can you add screenshot and follow the PR description?

File: app/src/main/java/com/example/cinescope/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            }
+        }
+    }
+}
+
+@Composable
+fun CineScopeApp() {
+    val navController = rememberNavController()
+
+    NavHost(navController = navController, startDestination = "home_screen") {
```
Comment (anandwana001) [inline app/src/main/java/com/example/cinescope/MainActivity.kt:34]: why hard code string?

File: app/src/main/java/com/example/cinescope/HomeScreen.kt [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+import androidx.compose.ui.unit.dp
+import com.example.cinescope.ui.theme.CineScopeTheme
+
+@Composable
+fun HomeScreen(
+    modifier: Modifier = Modifier,
+    navigateToDetailsScreen: () -> Unit
+) {
+    Column(
+        modifier = Modifier
```
Comment (anandwana001) [inline app/src/main/java/com/example/cinescope/HomeScreen.kt:40]: why not using the modifier coming in function params?

File: app/src/main/java/com/example/cinescope/BottomBar.kt [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun BottomBar() {
+    Row(
```
Comment (anandwana001) [inline app/src/main/java/com/example/cinescope/BottomBar.kt:19]: Why didn't we use bottom bar by material design?

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/cinescope/MainActivity.kt (hunk 1 above)
Comment (pr-Dutta) [reply app/src/main/java/com/example/cinescope/MainActivity.kt:23]: ok, I will follow the PR description.

File: app/src/main/java/com/example/cinescope/MainActivity.kt (hunk 2 above)
Comment (pr-Dutta) [reply app/src/main/java/com/example/cinescope/MainActivity.kt:34]: those are the temporary strings, I will change those.

File: app/src/main/java/com/example/cinescope/HomeScreen.kt (hunk 3 above)
Comment (pr-Dutta) [reply app/src/main/java/com/example/cinescope/HomeScreen.kt:40]: ok, Do I need to pass the modifier parameter from the parent in all cases.

File: app/src/main/java/com/example/cinescope/BottomBar.kt (hunk 4 above)
Comment (pr-Dutta) [reply app/src/main/java/com/example/cinescope/BottomBar.kt:19]: ok, let me change.

**Comment by anandwana001** [comment]
@pr-Dutta Day 1 Task Pending

**Comment by anandwana001** [comment]
Closing the PR as the challenge is completed.

**Pull request closed without merging**